src/main/java/net/dzikoysk/funnyguilds/FunnyGuilds.java

```diff
@@ -30,9 +30,13 @@
 import org.bukkit.plugin.java.JavaPlugin;
 
 import java.io.InputStream;
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
 
 public class FunnyGuilds extends JavaPlugin {
 
+    public static final DateFormat DATE_FORMAT = new SimpleDateFormat("dd.MM.yyyy HH:mm:ss");
```

Comment:
Format daty powinno się pobierać z pliku wiadomości bo jest on zależny od języka którym się posługujesz.

Reply:
To się doda kiedyś do konfiguracji